src/app/tecnologia/tecnologia.component.ts

```diff
@@ -0,0 +1,37 @@
+import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
+import { Component,} from '@angular/core';
+import { catchError, map, Observable, of } from 'rxjs';
+import { DashboardTecnologia } from '../modelosInterface/dashboardTecnologia';
+import { AutenticacaoFirebaseService } from '../servicosInterface/autenticacao-firebase.service';
+import { DashboardTecnologiaService } from '../servicosInterface/dashboard.tecnologia.service';
+
+@Component({
+  selector: 'app-tecnologia',
+  templateUrl: './tecnologia.component.html',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
  templateUrl: "./tecnologia.component.html",
```